atlp-rwanda/atlp-devpulse-fn
# #219 Highlight active sidebar item

### PR Description
This PR Introduces functionality of an Active Sidebar item to be highlighted

### Description of tasks that were expected to be completed
- Allow Active sidebar item to be highlighted

### How has this been tested?
- Clone repo
- Checkout branch `highlight-sidebarItems`
- Make sure that you set the environment variables referring to `.env.example`
- Run `npm install`, then `npm run dev` to check if the app is running successfully

With deployed Link: [Preview](https://vercel.live/open-feedback/atlp-devpulse-fn-git-highlig-48e089-devpulses-projects-62c56cf1.vercel.app?via=pr-comment-visit-preview-link&passThrough=1)

### Credentials
Email: `admin@example.com`
Password: `password123`

### Track PR (issue number & link)
[#219 ](https://github.com/atlp-rwanda/atlp-devpulse-fn/issues/219)

### Screenshots

How It now looks with active item highlighted
![Screenshot (55)](https://github.com/user-attachments/assets/dee3faaa-f014-46d1-b1a6-5e5c21ea82df)

![Screenshot (56)](https://github.com/user-attachments/assets/defdfeb7-7e86-4ff5-b4f2-b6b4f2d14507)
